tests/fixtures/commands/component-check.yml

```diff
@@ -41,7 +41,14 @@
         ======================================
 
         The use of drupal/codesnippet:1.8 is restricted. Contact QA Team.
+        Package drupal/core:9.4.8 has not been reviewed by QA.
```

Comment:
Include module evaluation for drupal/core and drupal/scaffol,....

Check the possibility to improve to test to check the presence of a substring instead of exact text.